react-router/react-router-tests.tsx

```diff
@@ -10,8 +10,27 @@ import * as ReactDOM from "react-dom"
 
 import { browserHistory, hashHistory, Router, Route, IndexRoute, Link } from "react-router"
 
+interface MasterContext {
+    router: ReactRouter.RouterOnContext;
+}
+
 class Master extends React.Component<React.Props<{}>, {}> {
 
+    static contextTypes: React.ValidationMap<any> = {
+        router: React.PropTypes.object
+    };
+    context: MasterContext;
+
+	navigate() {
```

Comment:
Ya, this file uses tabs, and my editor was inserting spaces.  Fixed it now and replaced the commit.